docs/device-agent/quickstart.md

```diff
@@ -32,8 +32,12 @@ _NOTE: If you do not have access to a terminal, follow the [Quick Start with Web
 2. Paste the command copied from the previous step and run it.
 
     <img src="images/device_cli.png" />
+3. Start the Device Agent using the command instructed by the device agent output e.g:
+   ```bash
+   flowfuse-device=agent -d /opt/flowfuse-device
```

Comment:
Type here with a `=` instead of `-`, also not sure we need to overcomplicate the command with `-d`, no need to include it, if they're configuring with differences it'll be in the docs accordingly

Reply:
> also not sure we need to overcomplicate the command with -d, no need to include it, if they're configuring with differences it'll be in the docs accordingly

Actually, I think that was down to my system having the old `flowfuse-device` directory - the OTC is clever enough to let you know that you MUST include that part (I edited the image manually)

I will do a full device add once more, on a clean system and put updated graphic and command in place

Reply:
Resolved in [7a13d51](https://github.com/FlowFuse/flowfuse/pull/3541/commits/7a13d51575341a8334e0c48ec26c181c45a8d1b1)